src/auto/style_context.rs

```diff
@@ -180,6 +183,8 @@ pub trait StyleContextExt: 'static {
 
     //fn get_style(&self, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs);
 
+    fn get_style_property(&self, property_name: &str, value: &mut glib::Value);
```

Comment:
Ah that's why it was ignored probably... this should *return* an `Option<glib::Value>`

Reply:
Not sure why gir gets the other function right and this one not though...

Reply:
@EPashkin any guess? :)

Reply:
IMHO because parameter not marked as `out`
```
          <parameter name="value" transfer-ownership="none">
            <doc xml:space="preserve" filename="gtk/gtkstylecontext.c" line="1727">Return location for the property value</doc>
            <type name="GObject.Value" c:type="GValue*"/>
          </parameter>
```

Reply:
That could be fixed with some addition to `gir-files/fix.sh` :)